Sources/IBLinterFrontend/Commands/ValidateCommand.swift

```diff
@@ -8,75 +8,50 @@
 import Foundation
 import IBDecodable
 import IBLinterKit
-import Commandant
+import ArgumentParser
 
-struct ValidateCommand: CommandProtocol {
-    typealias Options = ValidateOptions
-    typealias ClientError = Options.ClientError
+struct ValidateCommand: ParsableCommand {
+    static let configuration = CommandConfiguration(commandName: "lint", abstract: "Print lint warnings and errors")
 
-    let verb: String = "lint"
-    var function: String = "Print lint warnings and errors (default command)"
+    @Option(name: .long, help: "validate project root directory", completion: .directory)
+    var path: String?
+    @Option(name: .long, help: "the reporter used to log errors and warnings")
+    var reporter: String?
+    @Option(name: .long, help: "custom IBLinterfile.swift", completion: .file(extensions: ["swift"]))
+    var iblinterFilePath: String?
```

Comment:
This option is no longer supported, so could you remove it?

Reply:
Removed at 44d2d50